packages/core/src/components/graphs/scatter.ts

```diff
@@ -365,10 +365,30 @@ export class Scatter extends Component {
 					})
 					.classed('unfilled', false);
 
-				// Show tooltip for single datapoint
+				// Show tooltip
+				const bubbleOptions = Tools.getProperty(
+					self.getOptions(),
+					'bubble'
+				);
+
 				self.services.events.dispatchEvent(Events.Tooltip.SHOW, {
 					hoveredElement,
 					data: [datum],
+					additionalItems: [
+						{
+							label: Tools.getProperty(
+								bubbleOptions,
+								'radiusLabel'
+							),
+							value:
+								datum[
+									Tools.getProperty(
+										bubbleOptions,
+										'radiusMapsTo'
+									)
+								],
+						},
+					],
 				});
```

Comment:
if this only relates to bubble, then this should happen inside `bubble.ts` not `scatter.ts`

Reply:
`bubble.ts` doesn't have `addEventListeners()`, in order to put it in `bubble.ts` I need to override the function by copying the whole content and add `additionalItems` there.